services/resource-getter.js

```diff
@@ -19,10 +23,20 @@ function ResourceGetter(model, params) {
   }
 
   this.perform = function () {
+    var where = {};
+    if (schema.isCompositePrimary && params.recordId) {
+      var recordId = params.recordId.split('-');
+      if (recordId.length === _.keys(model.primaryKeys).length) {
+        _.keys(model.primaryKeys).forEach(function (key, index) {
+          where[key] = recordId[index];
+        });
+      } else { return; }
+    } else {
+      where.id = params.recordId;
```

Comment:
According to wikipedia, the wording you can use is "composite key" or "composite primary key" (not "primary composite key").